google-cloud-bigtable/pom.xml

```diff
@@ -376,6 +371,24 @@
           <trimStackTrace>false</trimStackTrace>
         </configuration>
       </plugin>
+
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.5.1</version>
+        <configuration>
+          <source>1.7</source>
+          <target>1.7</target>
+          <encoding>UTF-8</encoding>
+          <compilerArgument>-Xlint:unchecked</compilerArgument>
+          <annotationProcessorPaths>
+            <path>
+              <groupId>com.google.auto.value</groupId>
+              <artifactId>auto-value</artifactId>
+              <version>${autovalue.version}</version>
+            </path>
+          </annotationProcessorPaths>
+        </configuration>
```

Comment:
This causes build issues in IntelliJ. See the attached screenshot
![Screenshot from 2019-12-02 12-13-20](https://user-images.githubusercontent.com/23526194/69979955-43808900-14fd-11ea-97ac-96bf958946f0.png)

Reply:
Actually nevermind, it fails with annotation processing enabled as well

Reply:
It seems like intellij is failing to resolve transitive deps for `annotationProcessorPaths`, explicitly adding auto-service alongside auto-value fixes the intellij issue:

```
<plugin>
        <artifactId>maven-compiler-plugin</artifactId>
        <version>3.5.1</version>
        <configuration>
          <source>1.7</source>
          <target>1.7</target>
          <encoding>UTF-8</encoding>
          <compilerArgument>-Xlint:unchecked</compilerArgument>
          <annotationProcessorPaths>
            <path>
              <groupId>com.google.auto.value</groupId>
              <artifactId>auto-value</artifactId>
              <version>${autovalue.version}</version>
            </path>
            <path>
              <groupId>com.google.auto.service</groupId>
              <artifactId>auto-service</artifactId>
              <version>1.0-rc6</version>
            </path>
          </annotationProcessorPaths>
        </configuration>
      </plugin>
```

But we would have to manually keep the 2 versions in sync. I don't think its worth the added maintenance cost